extensions/jupiter-aggregator/src/menu-bar-token-price.tsx

```diff
@@ -0,0 +1,22 @@
+import { MenuBarExtra } from "@raycast/api";
+import { fetchPrice } from "./api";
+
+export default function Command() {
+  const { data, isLoading } = fetchPrice("SOL");
+  const price = data ? parseFloat(data.data.SOL.price).toFixed(2) : "loading";
+
+  return (
+    <MenuBarExtra
+      icon={{
+        source: "solana.svg",
+        tintColor: {
+          light: "black",
+          dark: "white",
+        },
+      }}
+      tooltip="gm"
+      title={data ? `$${price}` : "loading"}
```

Comment:
While it's loading, I think we should do an early `null` return, instead of changing the title to `loading`. It can be quite jarring otherwise to see things flicking in the Menu Bar